initfiles/componentfiles/configschema/xsd/ldapserver.xsd

```diff
@@ -42,6 +42,10 @@
                               hpcc:tooltip="Time in minutes after which the cached security information should expire"/>
                 <xs:attribute name="sharedCache" type="xs:boolean" hpcc:displayName="Shared Cache" hpcc:presetValue="true"
                               hpcc:tooltip="Use a single, shared LDAP cache"/>
+                <xs:attribute name="systemUserVaultID" type="xs:string" hpcc:displayName="System User Vault ID"
```

Comment:
I may have missed something, but these two values need to be settable through the helm chart.  
Well actually we have 2 choices... 1. use a well known name as the secret name and search for it in our new category.  or 2. add helm configuration as you've done here for configmgr.

If you want to use a well defined name like for example "ldapSystemUser" under the new secrets category of "authn" then you just have to change your call to getSecret below to:

secretTree.setown(getSecret("authn", "ldapSystemUser"));

That method will search the category local secrets and vaults to find the secret with that name.